<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginManager</class>
 <widget class="QDialog" name="PluginManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plugin Manager</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="pluginTable">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="enableButton">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disableButton">
       <property name="text">
        <string>Disable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="configureButton">
       <property name="text">
        <string>Configure</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="installButton">
       <property name="text">
        <string>Install Plugin</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 